locksmith/__tests__/utils/lockIcon.test.ts

```diff
@@ -0,0 +1,11 @@
+import { lockIcon } from '../../src/utils/lockIcon'
+
+describe('lockIcon', () => {
+  it('should generate the right svg for different locks', () => {
+    expect.assertions(2)
+    const lockIcon1 = lockIcon('0xAaAdEED4c0B861cB36f4cE006a9C90BA2E43fdc2')
+    const lockIcon2 = lockIcon('0x95de5F777A3e283bFf0c47374998E10D8A2183C7')
+    expect(lockIcon1).toMatch(/^<svg/)
```

Comment:
Not a blocker but we may want to see if we can add a bit better criteria to match up on here. I think an empty svg may end up sliding though this test.

Reply:
Yeah, that's a fair point. I am testing that it's different for 2 lock addresses though so that should probably at least make sure that this code does something...